Jenkinsfile

```diff
@@ -0,0 +1,142 @@
+#!groovy
```

Comment:
for Android we need to have access to the device to run tests inside, currently this is how `realm-java` is setup https://github.com/realm/realm-java/blob/master/Jenkinsfile#L43

The current android test also extract a file `tests/react-test-app/tests.xml` that the current conf in CI uses to publish Unit Test result
<img width="730" alt="screen shot 2016-11-23 at 18 44 09" src="https://cloud.githubusercontent.com/assets/1793238/20574546/e050da62-b1ac-11e6-9799-6be442ffc3a8.png">